src/services/WolfDen.Infrastructure/Configuration/LeaveBalanceConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using WolfDen.Domain.Entity;
+
+namespace WolfDen.Infrastructure.Configuration
+{
+    public class LeaveBalanceConfiguration:IEntityTypeConfiguration<LeaveBalance>
+    {
+        public void Configure(EntityTypeBuilder<LeaveBalance> builder)
+        {
+            builder.HasOne(x => x.Employee).WithMany().HasForeignKey(x => x.EmployeeId);
```

Comment:
add if anything is required make it as required

Reply:
Required fields are specified in configurations